test/unit/lib/postcode_sanitizer_test.rb

```diff
@@ -0,0 +1,22 @@
+require 'test_helper'
+
+class PostcodeSanitizerTest < ActiveSupport::TestCase
+
+  context "postcodes come through to imminence" do
+
+    should "strip trailing spaces from entered postcodes" do
+      assert_equal "WC2B 6NH", PostcodeSanitizer.sanitize("WC2B 6NH ")
+    end
+
+    should "strip non-alphanumerics from entered postcodes" do
+      assert_equal "WC2B 6NH", PostcodeSanitizer.sanitize("WC2B   -6NH]")
+    end
+
+    should "transpose O/0 and L/1 if necessary" do
```

Comment:
Oops, fixed and amended the commit message. That was probably from my brain wondering "it would be nice if this worked for the various different typos of my online identity". :stuck_out_tongue: